### ISSUE TITLE
[2021 fall] CVPR 2017 Video Frame Interpolation via Adaptive Convolution (20213472)

### ISSUE THREAD

**Issue by niceDuckgu** [awesome-davian/awesome-reviews-kaist#167]

**Comment by jhjun724** [comment]
안녕하세요. 먼저 좋은 논문 리뷰 감사합니다.
굉장히 간결하고 명료한 설명 덕분에 쉽게 이해가 되었고,
리뷰하신 논문을 깊게 읽어보진 않았지만 논문을 읽은 것만큼 많은 공부가 되었던 것 같습니다.

먼저 리뷰를 읽으며 생긴 논문에 대한 저의 개인적인 궁금증을 풀어주셨으면 싶어서 코멘트를 남깁니다.

1. 네트워크에 input에 해당하는 두 receptive field는 depth 방향으로 concatenate 된 반면에, output feature와 그와 convolution될 input patch는 horizontal 방향으로 concatenate 된 느낌이었습니다. 제가 맞게 이해했다면 output feature와 input patch가 convolution 되었을 때 최종 output으로 RGB 값, 즉 3채널의 결과를 얻기위해서 input patch를 horizontal한 방향으로 concatenate하여 3채널을 유지한 것 외에 다른 이유가 있는지, receptive field 또한 horizontal 방향으로 concatenate 한다면 어떤 점이 달라질지 궁금합니다.

2. 첫번째 질문에서 output feature가 concatenate 됐다 표현했는데 엄밀히 이는 옳지 않은 표현이라는 것을 알고 있습니다. Convolution Block이 어떤 layer들로 구성되어 있는지를 보여주는 Table 1.을 보았을 때 softmax까지 거친 3362x1x1 feature가 input patch (41x82x3)와 convolution 된다는게 정확히 맞는 표현이겠지만, 3362x1x1이 어떻게 41x82로 변환되는 중간과정이 필요할 것이라 생각됩니다. 그 변환 과정이 어떻게 되는지 궁금합니다.

3. 새로운 피사체의 유입 혹은 카메라의 움직임으로 인해, 카메라의 가장자리일수록 에러가 클 것이라고 예상이 됩니다. 제 예상이 맞는지 맞다면 그에 대한 분석도 있는지, 혹은 이러한 상황은 occlusion 경우와 비슷한 경우로 취급하여 따로 추가되는 분석은 없는지 궁금합니다.

위의 질문에 대한 답변들이 최종 paper review에 포함되면 좋을 것 같습니다. 제 질문 자체가 두서가 부족한 것 같아, 혹시 코멘트가 이해 안되는 부분이 있으시다면 다시 정리해 코멘트 남기도록 하겠습니다. 

추가적으로 오타와 같은 수정사항에 대해 코멘트 남깁니다.
1. Method에서 Convolution kernel estimation 부분의 Table 1의 타이틀이 다른 Figure 혹은 Table의 title 포맷과 다른 것 같습니다.

다시 한번 좋은 리뷰 감사합니다.

**Comment by LimGyeongrok** [comment]
안녕하세요 좋은 리뷰 잘 읽어보았습니다:)
리뷰를 읽고 몇 가지 코멘트를 남기고자 합니다.

1. Loss function 부분에 나온 식이 중간에 잘려 다음 줄로 넘어간 것으로 보이는데 loss function이 한 줄로 나와있으면 좀 더 좋을 것 같습니다.
2. 해당 네트워크 과정의 inference time을 알 수 있을까요?
3. Reviewer information이 누락되어 있는 것 같습니다.

감사합니다.

**Comment by oglee815** [comment]
20213784 / 이현제

안녕하세요. 정성들인 리뷰 잘 읽었습니다.
DNN으로 Video frame을 interpolation하는 것에 대해 매우 흥미로웠습니다.

추가적으로,
제안하는 방식이 일부 데이터 셋에서 좋은 성능을 보이지 못했는데 이에 대해서 'network가 real-word scene의 image들로 training을 하기 때문'이라고 적혀 있는데, 이 부분이 왜 성능에 안좋은 영향을 끼쳤는지 추가 설명이 있으면 좋을 것 같습니다.

감사합니다.
